tests/manage/pv_services/test_del_mon_service_and_create_pvc.py

```diff
@@ -0,0 +1,224 @@
+import json
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    skipif_external_mode,
+    E2ETest,
+    tier4a,
+    ignore_leftovers,
+    bugzilla,
+)
+from ocs_ci.helpers.helpers import modify_deployment_replica_count
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP, get_services_by_label
+from ocs_ci.ocs.resources.pod import (
+    get_mon_pods,
+    get_operator_pods,
+    run_io_in_bg,
+    wait_for_storage_pods,
+)
+from ocs_ci.utility.utils import ceph_health_check
+
+log = logging.getLogger(__name__)
+
+POD_OBJ = OCP(kind=constants.POD, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE)
+
+
+@bugzilla("1858195")
+@tier4a
+@ignore_leftovers
+@skipif_external_mode
+@skipif_ocs_version("<4.7")
```

Comment:
The fix is in 4.6.3, should we run even on 4.6.3?
I think you can make use of semantic_version here, please check

Reply:
This is the proper way to use skipif_ocs_version as we evaluate this in that function and this is expected. The function itself should be later on updated to use semantic version which we currently do not I think.
We can use only x.y version in this function as we don't specify z in ocs_version in config.

If that's the case and it's fixed in 4.6.3 and we already have 4.6.4 we should have:
```
@skipif_ocs_version("<4.6")
```

Reply:
ack. @Akarsha-rai Please add "@skipif_ocs_version("<4.6")" so that the test can run even on 4.6 latest.